src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -230,24 +237,48 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
                 label={t('registration.files_and_license.conditions_for_using_file')}
                 required
                 onChange={({ target: { value } }) => setFieldValue(field.name, value)}>
-                {licenses
-                  .filter((license) => license.version === 4 || !license.version)
-                  .map((license) => (
-                    <MenuItem
-                      data-testid={dataTestId.registrationWizard.files.licenseItem}
-                      key={license.id}
-                      value={license.id}
-                      divider
-                      dense
-                      sx={{ gap: '1rem' }}>
-                      <ListItemIcon>
-                        <img style={{ width: '5rem' }} src={license.logo} alt={license.name} />
-                      </ListItemIcon>
-                      <ListItemText>
-                        <Typography>{license.name}</Typography>
-                      </ListItemText>
-                    </MenuItem>
-                  ))}
+                {activeLicenses.map((license) => (
+                  <MenuItem
+                    data-testid={dataTestId.registrationWizard.files.licenseItem}
+                    key={license.id}
+                    value={license.id}
+                    divider
+                    dense
+                    sx={{ gap: '1rem' }}>
+                    <ListItemIcon>
+                      <img style={{ width: '5rem' }} src={license.logo} alt={license.name} />
+                    </ListItemIcon>
+                    <ListItemText>
+                      <Typography>{license.name}</Typography>
+                    </ListItemText>
+                  </MenuItem>
+                ))}
+                <MenuItem
+                  data-testid={dataTestId.registrationWizard.files.licenseItemShowOlderVersion}
+                  title={t('registration.files_and_license.show_all_older_versions')}
+                  sx={{ display: 'flex', justifyContent: 'center' }}
+                  onClickCapture={(e) => {
+                    e.stopPropagation();
+                    setInactiveLicensesOpen(!inactiveLicensesOpen);
+                  }}>
+                  <MoreHorizIcon />
+                </MenuItem>
+                {inactiveLicenses.map((license) => (
+                  <MenuItem
+                    data-testid={dataTestId.registrationWizard.files.licenseItem}
+                    key={license.id}
+                    value={license.id}
+                    divider
+                    dense
+                    sx={{ gap: '1rem', display: inactiveLicensesOpen ? 'flex' : 'none' }}>
+                    <ListItemIcon>
+                      <img style={{ width: '5rem' }} src={license.logo} alt={license.name} />
+                    </ListItemIcon>
+                    <ListItemText>
+                      <Typography>{license.name}</Typography>
+                    </ListItemText>
+                  </MenuItem>
+                ))}
```

Comment:
Prøvde på å trekke ut dette til en felles komponent, men ble bare mye kluss da det er mange parametre som må sendes inn, og options må være likt strukturert for at verdier skal kunne velges (når det trekkes ut til en komponent så aksepteres ikke opsjonene lengre sånn uten videre).